README.md

```diff
@@ -3,9 +3,20 @@
 `mktrayicon` is a simple proxy program that lets you create and modify system
 tray icons without having to deal with a graphical toolkit like GTK.
 
-`mktrayicon` expects to be given a name pipe (FIFO) file path when it is
-started, and you control your status icon by writing to this named pipe. *Note
-that the pipe should already be created before you call `mktrayicon`*.
+`mktrayicon` can be used two ways: To create an icon that is controlled by a named pipe
+or, more simply, to create a non-interactive icon using the specified tooltip string (which can be an empty string if you don't need a tooltip). 
+
+In all, there are three ways of calling mktrayicon:
+```
+$ mktrayicon <FIFO> &
+  or
+$ mktrayicon -i <ICON> <FIFO> &
+  or
+$ mktrayicon -i <ICON> -t <TOOLTIP> &
+```
```

Comment:
Is
```console
$ mktrayicon -i <ICON> &
```
supported too?

Reply:
No, but we can do that. I just didn't want to change things around too much.